rusk-wallet/src/bin/command.rs

```diff
@@ -191,11 +191,11 @@ pub(crate) enum Command {
 
         /// Function name to call
         #[clap(short, long)]
-        fn_name: String,
+        name: String,
 
         /// Function arguments for this call
-        #[clap(short, long)]
-        fn_args: Vec<u8>,
+        #[clap(short = 'f', long)]
+        param: Vec<u8>,
```

Comment:
```suggestion
        args: Vec<u8>,
```
I would name this arg or args. The doc comment above still says arguments, and there is a semantic difference between parameter & argument.